packages/js/product-editor/src/components/variations-table/variations-table.tsx

```diff
@@ -496,6 +497,17 @@ export const VariationsTable = forwardRef<
 									</div>
 								</Tooltip>
 							) }
+
+							<Button
+								href={ getNewPath(
+									{},
+									`/product/${ productId }/variation/${ variation.id }`,
+									{}
+								) }
```

Comment:
Is there a way to make this use the react router when redirecting?
It seems like the `href` is causing it to do a full page refresh, which seems unnecessary.
I am not sure why React Router isn't catching this.